.travis.yml

```diff
@@ -169,11 +176,21 @@ install:
         pip install -q PySDL2;
       fi;
 
+    # Install OSMesa
+    - wget https://github.com/julienr/vispy_osmesa_docker_12_04/releases/download/0.1/osmesa.tar.bz2 -O /tmp/osmesa.tar.bz2
```

Comment:
Can you add a PR to this repo to make an `osmesa` directory with this tar inside it:

https://github.com/vispy/demo-data

Might want to call it `osmesa/osmesa_VERSON_12_04.tar.bz2` or so.